Add shared infrastructure and add communications to team guide (#451)
jupyterhub/team-compass: docs/team/shared-infrastructure.md

# Our Shared Infrastructure

There are a few services and infrastructure that the JupyterHub team has access to.
Some of these are useful for our development workflows, while others are used as a part of services that we run.

## GitHub

We have [a GitHub organization](https://github.com/jupyterhub/) for hosting all of our code repositories.
This organization is where we do most of the code-related work for the project, and where we have discussions and coordination.

## Image registry on quay.io

We have [a quay.io organization](https://quay.io/organization/jupyterhub) for hosting Docker images.
This is used for the repo2docker base image.
If you do not have access to this organization and require it, ask a team member to add you.

## Image registry on Dockerhub

We have [a DockerHub organization](https://hub.docker.com/r/jupyterhub/jupyterhub/) for hosting Docker images.

> BinderHub and JupyterHub images are pushed to DockerHub as that still works, though we are considering publishing these images to use quay.io as well.

Author: ```suggestion
BinderHub and JupyterHub images are pushed to DockerHub as that still works, though we are considering publishing these images to use quay.io as well.
If you do not have access to this organization and require it, ask a team member to add you.
```

# Our Shared Infrastructure

There are a few services and infrastructure that the JupyterHub team has access to.
Some of these are useful for our development workflows, while others are used as a part of services that we run.

## GitHub

We have [a GitHub organization](https://github.com/jupyterhub/) for hosting all of our code repositories.
This organization is where we do most of the code-related work for the project, and where we have discussions and coordination.

## Image registry on quay.io

We have [a quay.io organization](https://quay.io/organization/jupyterhub) for hosting Docker images.

> This is used to push new images for tools like BinderHub and JupyterHub for Kubernetes.

Reviewer: It's a bit more complicated at the moment. Current situation:
- repo2docker: pushed to quay.io only due to problems with accessing the jupyter Docker Hub account
- BinderHub and JupyterHub: only pushed to Docker Hub as that still works, but I think we should switch to pushing releases to both quay.io and Docker Hub, and dev images to quay.io.

> If you do not have access to this organization, ask a team member to add you.

Reviewer: ```suggestion
If you do not have access to this organization and require it, ask a team member to add you.
```

> We have [a quay.io organization](https://quay.io/organization/jupyterhub) for hosting Docker images.
> This is used to push new images for tools like BinderHub and JupyterHub for Kubernetes.
> If you do not have access to this organization and require it, ask a team member to add you.

Author: ```suggestion
We have [a quay.io organization](https://quay.io/organization/jupyterhub) for hosting Docker images.
This is used for the repo2docker base image.
If you do not have access to this organization and require it, ask a team member to add you.
```

> ## Image registry on Dockerhub

Reviewer 2: ```suggestion
## Image registry on DockerHub
```

> We have [a DockerHub organization](https://hub.docker.com/r/jupyterhub/jupyterhub/) for hosting Docker images.
> This was used more heavily in the past, but due to recent DockerHub pricing changes it is no longer a sustainable option.
> However, we're listing it here since some of our images are still hosted there.
> 
> ## Google Cloud Project

Author: ```suggestion
We have [a DockerHub organization](https://hub.docker.com/r/jupyterhub/jupyterhub/) for hosting Docker images.
BinderHub and JupyterHub images are pushed to DockerHub as that still works, though we are considering publishing these images to use quay.io as well.

## Google Cloud Project
```
